docs/API.md

```diff
@@ -187,12 +186,18 @@ When a method received an unknown *profile name* value for in parameter with thi
 
 ```json
 {
-    "jsonrpc": "2.0",
-    "id": 1,
-    "result": {
-        "message": "Invalid profile option format",
-        "status": "nok"
-    }
+  "error": {
+    "code": "-32602",
+    "data": [
+      {
+        "path": "/profile",
+        "message": "Unknown profile"
+      }
+    ],
+    "message": "Invalid method parameter(s)."
+  },
+  "id": 1,
+  "jsonrpc": "2.0"
```

Comment:
This should be included with the error message below, shouldn't it?

Reply:
I'm not super comfy about detailing the behavior of methods in the data type definitions. E.g. what if you have two methods that take a profile argument, but in different locations of their arguments structures? I think it's better to let each method list its own error modes. Though in the case of validation errors it would be better with a generic description of how the validation errors are structured (with paths and messages etc) in the error response.

Reply:
This example response has been removed